docs/baking.md

```diff
@@ -0,0 +1,185 @@
+package. 
+
+However, by default, these services will start to bootstrap the node from scratch,
+which will take a significant amount of time.
+In order to avoid this, here we suggest bootstrapping from a snapshot instead.
+
+### Setting up node data directory
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as its RPC address by default.
+These parameters can be changed in the service configuration, which is located in
+`/lib/systemd/system/tezos-node-<network>.service`. Note, that if you've updated the `.service`
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
+
+### Bootstrapping the node
+
+In order to run a baker locally, you'll need a fully-synced local `tezos-node`.
+
+The fastest way to bootstrap the node is to import a snapshot.
+Snapshots can be downloaded from https://snapshots.tulip.tools/#/ .
+
+All commands within the service are run under the `tezos` user.
+
+Download the snapshost for desired network and with the desired history mode
+(you can read more about `tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+Snapshot import requires that the `tezos-node` data directory doesn't contain the `context` and `store` folders.
+Remove them before the snapshot import.
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+In case you're getting an error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do that run one of the following commands
+depending on whether the config in the given data directory was initialized previously:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+#or
+sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
+```
+
+### Starting the node
+
+After the snapshot import, you can finally start the node by running
+```
+sudo systemctl start tezos-node-<network>.service
+```
+
+Note that even after the snapshot import the node can still be out of sync, it may require
+some additional time to completely bootstrap.
+
+To stop the node run:
+```
+sudo systemctl stop tezos-node-<network>.service
+```
+
+You can check the node logs via `journalctl`:
+```
+journalctl -u tezos-node-<network>.service
+```
+
+## Setting up baker and endorser daemons
+
+Once you have fully synced your local `tezos-node`, you can start the baker and endorser daemons.
+
```

Comment:
do you actually have to wait for the node to fully sync or can you start baker and endorser and they will figure out when the node is ready themselves?

Reply:
You can indeed start both daemons, but they won't do anything unless the node is completely bootstrapped

Reply:
_unless_ or _until_? if user can set everything up upfront and later just needs to check  that everything looks good once the node is synced is less mental effort than pausing the entire set up until node is bootstrapped and resuming it later, having to remember where were you in this complex workflow

Reply:
> unless or until?

:facepalm: 
of course, until